common.py

```diff
@@ -113,8 +114,9 @@ async def kick_message(message: Message, context: CallbackContext, db: database.
 	# get rid of deleted messages in memory so we can remember more potentially important messages
 	remove_from_recent_messages(*todel)
 	try:
-		if message.text is not None and len(message.text) >= CONFIG['spam_minlength']:
-			thisdigest = hashdigest(message.text)
+		urls = re.findall(r"https?://[^\s]+", update.message.text)
```

Comment:
probably works as a naïve approach, but can we try searching for [`telegram.MessageEntity`](https://docs.python-telegram-bot.org/en/stable/telegram.messageentity.html#telegram.MessageEntity) in `message.entities` instead?